src/content/docs/style-guide/word-choice/usage-dictionary.mdx

```diff
@@ -374,6 +378,20 @@ For consistency across New Relic content, here are our general word usage guidel
     Unlike the Latin abbreviations [`e.g.`](#eg) or [`i.e.`](#ie)), you're welcome to use etc. It should have a period at the end (`etc.`). Ensure that you have several meaningful examples, though, before using it. For example, `cities including Portland, Seattle, Dublin, etc.` but not `cities including Portland, etc.`.
   </Collapser>
 
+    <Collapser
+    id="event"
+    title="event"
+  >
+To understand how to optimally write about events, we recommend reading: 
+* The [event section in the data types doc](/docs/data-apis/understand-data/new-relic-data-types/#event-data)
+* [The glossary entry for "event"](/docs/glossary/glossary/#event)
+
+Some language-related considerations when writing about events: 
+* Prior to the roll-out of our three other core data types of `Metric`, `Log`, and `Span`, at New Relic we referred to all NRDB-stored objects as "events." For this reason, internally and sometimes externally we still sometimes refer to all NRDB objects as "events." (For example, we might simply say "event data" when what we mean is NRDB-stored data of any type. Or, in [the query UI](/docs/query-your-data/nrql-new-relic-query-language/get-started/rate-limits-nrql-queries/#inspected-count-limits), we refer to "events inspected" and this refers to all NRDB data types). So it's important to understand some of this ambiguity around how we use "event." For public resources, we should attempt to use "event" to refer to NRDB-stored data **that is not** `Metric`, `Log`, or `Span` data types. 
```

Comment:
This might be easier to digest by simplifying the paragraph. My instinct is to take the 'For example' part of the paragraph out of the parens, and make those 2 examples listed as bulleted points. 

How do you feel about moving this sentence, "It's important to understand some of this ambiguity around how we use "event."" to the beginning of the paragraph? 

As a newbie, having a preface about incoming ambiguity better prepares my brain for creating a mind map of something complicated. Then the rest of the paragraph moves linearly into defining the ambiguity, giving an example, then providing further clarity (your conclusion) with a use-case for when not to use the word 'event.'